### PR TITLE
Redesign fixes

### PR DESCRIPTION
**Pull request by ntrncic**

### CODE REVIEW

File: EquiTrack/funds/tests/test_views.py [hunk 1]
```diff
@@ -112,15 +112,12 @@ def test_get_fail_with_one_bad_value(self):
                          'One or more of the FRs selected is either expired, has been used by another '
                          'intervention or could not be found in eTools')
 
-    def test_get_fail_with_expired_fr(self):
+    def test_get_success_with_expired_fr(self):
```
Comment (robertavram) [inline EquiTrack/funds/tests/test_views.py]: @ntrncic  i think there is another validation that needs to be scrapped... give me 1 min

Comment (robertavram) [reply EquiTrack/funds/tests/test_views.py]: funds.views.FRsView... that's where the validation is executed... line 30... end_date_gte==today

Comment (ntrncic) [reply EquiTrack/funds/tests/test_views.py]: yes I changed that line and tested adding an expired FR.

Comment (robertavram) [reply EquiTrack/funds/tests/test_views.py]: blind today...

**Pull request merged**